examples/testing-dom__download/cypress/integration/spec.js

```diff
@@ -162,6 +170,15 @@ describe('file download', () => {
       cy.log('**confirm downloaded image**')
       validateImage()
     })
+
+    // in Firefox we get "what should I do with this download?" popup
+    it('ZIP archive', { browser: '!firefox' }, () => {
```

Comment:
I think you could do `browser.helperApps.neverAsk.saveToDisk` with the mimeType:

```js
module.exports = (on) => {
  on('before:browser:launch', (browser, options) => {
    if (browser.family === 'firefox') {
      // needed to prevent download prompt for zip files.
      options.preferences['browser.helperApps.neverAsk.saveToDisk'] = 'application/zip'

      return options
    }
  })
}
```

Reply:
You are right, and I already had code do this for CSV and Excel files, updated